src/Apps/MyCollection/myCollectionRoutes.tsx

```diff
@@ -0,0 +1,23 @@
+import loadable from "@loadable/component"
+import { AppRouteConfig } from "System/Router/Route"
+
+const MyCollectionArtworkApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "myCollectionBundle" */ "./Routes/MyCollectionArtwork/MyCollectionArtworkApp"
+    ),
+  {
+    resolveComponent: component =>
+      component.MyCollectionArtworkAppFragmentContainer,
+  }
+)
```

Comment:
```suggestion
const MyCollectionArtwork = loadable(
  () =>
    import(
      /* webpackChunkName: "myCollectionBundle" */ "./Routes/MyCollectionArtwork/MyCollectionArtwork"
    ),
  {
    resolveComponent: component =>
      component.MyCollectionArtworkAppFragmentContainer,
  }
)
```

Reply:
This is the pattern used in other apps. The routes inside don't get the `App` postfix.